.github/workflows/checks.yml

```diff
@@ -0,0 +1,54 @@
+name: cmake
+on: [check_run, pull_request]
+jobs:
+  cmake-build:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest]
+
+    steps:
+      - name: Checkout project
+        uses: actions/checkout@v4
+
+      - name: Cache vcpkg
+        uses: actions/cache@v4
+        with:
+          path: vcpkg
+          key: vcpkg-cache-${{ runner.os }}
+
+      - name: Apt install for Ubuntu
+        if: matrix.os == 'ubuntu-latest'
+        run: |
+          sudo apt install libxinerama-dev libxcursor-dev xorg-dev libglu1-mesa-dev pkg-config
```

Comment:
Why this for Ubuntu and not for Windows ?

Reply:
Because it is required by cmake on ubuntu